packages/front-end/pages/getstarted.tsx

```diff
@@ -17,6 +22,8 @@ const GetStartedPage = (): React.ReactElement => {
     experiments: ExperimentInterfaceStringDates[];
   }>(`/experiments`);
 
+  const { data } = useApi<{ metrics: MetricInterface[] }>(`/metrics`);
```

Comment:
You don't need an API call for this.  The `useDefinitions` hook makes metrics available anywhere throughout the app.

```tsx
const {metrics} = useDefinitions();
```

And because of that hook, you don't need to get the metrics data on this page and pass it down via props.  You can just use that hook in the exact component where it's needed